src/Driver/LazyLoadingDriver.php

```diff
@@ -21,12 +20,12 @@ class LazyLoadingDriver implements DriverInterface
     private $realDriverId;
 
     /**
-     * @param ContainerInterface|PsrContainerInterface $container
+     * @param ContainerInterface $container
      */
     public function __construct($container, string $realDriverId)
```

Comment:
Couldn't we just use a native php type hint here?

```suggestion
    public function __construct(ContainerInterface $container, string $realDriverId)
```

Reply:
Technically adding a typehint is a B/C break, though the typehint would match up with the runtime check in the constructor (it just changes the error type).  If it’s OK with the maintainers, we can add the typehint and remove the inline check.